src/Model/WidgetController.php

```diff
@@ -1,12 +1,11 @@
 <?php
 
-namespace SilverStripe\Widgets\Controllers;
+namespace SilverStripe\Widgets\Model;
```

Comment:
Why was this move done?
Naming convention sees it as an oxymoron :/

Reply:
The controller must be in the same namespace, see `Widget::getController` for the current lookup logic